src/user/user.c

```diff
@@ -35,8 +42,31 @@ int main(int ac, char **argv)
                 return 0;
         }
 
-        sleep(20);
+        // get the fd for the map created in the BPF program
+        map_fd = bpf_map__fd(skel->maps.output_map);
+
+        // count the total number of processes executed on the CPUs each second
+        for (int i = 0; i < 1000; i++) {
```

Comment:
This feels a bit arbitrary. Maybe I love looking at the output of this program and would be disappointed that after 1000 seconds it stops printing numbers. What about `while (1)` ?